### DISCUSSION TITLE
Oct. 8 Team Check-In

### DISCUSSION THREAD

**Discussion by Ksarrge** [ecn310/course-project-taxes-tariffs#7] [General]
welcome

**Comment by DylanTh20** [comment]
- I helped the team by working on the causal diagrams for our first and second research papers
- I could have checked the work we did as a group to ensure it was ready for submission
- We can improve by meeting more frequently to discuss ideas

**Reply by Ksarrge** [reply]
Very good points. I agree with your thoughts.

**Comment by Ksarrge** [comment]
- Something someone did to help the team this week: Umar found the 4th and 5th papers
- Something I could have done better for the team this week: Made sure I was on time to class
- An area/suggestion for team improvement: communicate if you are going to miss class

**Reply by DylanTh20** [reply]
I agree about communicating when you will miss class as that is the time we complete the most work for the project.
